examples/resources/commercetools_api_extension/resource.tf

```diff
@@ -1,6 +1,4 @@
 resource "commercetools_api_extension" "my-extension" {
-  key = "test-case"
-
```

Comment:
I think we should keep this one?

Reply:
I'll add `key` to all examples then. `key` to me sounds a bit confusing, because even though it's optional, you can't actually delete it once it is set. Actually it should be a good candidate for _computed optional_, right? When it's not set, it should use the existing value. That way even if you delete it, you won't get `Error: Invalid key ''. ` errors.